lib/companies/jobs.ex

```diff
@@ -86,6 +86,10 @@ defmodule Companies.Jobs do
       {:error, %Ecto.Changeset{}}
 
   """
+  def delete(%Job{} = job) do
+    Repo.delete(job)
+  end
+
```

Comment:
Would it be a good idea to add a `checker` user or something uniquely named which can indicate which one made the change, and as a result leave the api intact?

Reply:
I guess it would be nice idea to just implement soft-delete all over the app, this would simplify everything. Not so sure about how to store delete reasons. String i.e. "deleted by automatic url checker", "deleted by github user ____"?

Reply:
We could devise a way to leverage the PendingChange tracking (which has `note` and an associated `user`) for this with an auto-approval for changes submitted via this checker (and admins)?

Reply:
Yep, and also add a soft-delete from #488 as a safety net.